scanner/src/main/kotlin/scanners/fossid/FossId.kt

```diff
@@ -537,8 +537,12 @@ class FossId internal constructor(
         if (response.data?.status in SCAN_STATE_FOR_TRIGGER) {
             log.info { "Triggering scan as it has not yet been started." }
 
-            service.runScan(config.user, config.apiKey, scanCode, *runOptions)
-                .checkResponse("trigger scan", false)
+            val scanResult = service.runScan(config.user, config.apiKey, scanCode, *runOptions)
+            // Scans that were added to the queue are interpreted as an error by FossID. However, `waitScanComplete()`
```

Comment:
Nit: Blank line before this one to better group the comment to the code below than the code above?